server/src/internalClusterTest/java/org/opensearch/cluster/ClusterHealthIT.java

```diff
@@ -439,4 +442,130 @@ public void clusterStateProcessed(String source, ClusterState oldState, ClusterS
             completionFuture.actionGet(TimeValue.timeValueSeconds(30));
         }
     }
+
+    public void testHealthAtLevelParamEqualsClusterIsHonoured() {
```

Comment:
Lets add `AWARENESS_ATTRIBUTES` level IT as well .

Reply:
+1, since there are 4 levels.